### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by danewalton-msft**
This removes:
1. Sections in the table of contents that no longer exist.
2. Instruction to comment out the device id since that breaks compilation.

This also adds back the section giving more instruction on how to interact with a PnP device.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -528,6 +513,16 @@ Here are the steps:
 
         <img src=".//media/image35.png" style="width:5.38735in;height:3.18982in" alt="A screenshot of a cell phone Description automatically generated" />
 
+### Further instructions for IoT Plug and Play
+
+- For the IoT Plug and Play sample, download the IoT Plug and Play DTMI [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json).
```
Comment (ericwolz) [inline README.md:518]: Will this be obsolete once microchip submits their changes?

File: README.md [hunk 2]
```diff
@@ -528,6 +513,16 @@ Here are the steps:
...
+- Once you open the IoT Explorer tool, on the left side, there should be a tab called "IoT Plug and Play Settings". Open that tab and add the directory where you downloaded the DTMI detailed from [prerequisites](#prerequisites).
```
Comment (ericwolz) [inline README.md]:
```suggestion
- Once you open the IoT Explorer tool, on the left side, there will be a tab called "IoT Plug and Play Settings". Open that tab and add the directory where you downloaded the DTMI detailed from [prerequisites](#prerequisites).
```

File: README.md [hunk 3]
```diff
@@ -528,6 +513,16 @@ Here are the steps:
...
+- Paste in the connection string for your IoT Hub from the portal.
```
Comment (ericwolz) [inline README.md]:
```suggestion
- Paste in the connection string for your IoT Hub from the Azure portal.
```

File: README.md [hunk 4]
```diff
@@ -528,6 +513,16 @@ Here are the steps:
...
+- Find your device listed after clicking on your newly connected IoT Hub and navigate to the left hand side where you will find a "IoT Plug and Play components" tab.
+- There, click on the component listed near the bottom and use the tabs that result on the top to use the various features (Commands, Twin, Telemetry). It should look similar to the picture below.
```
Comment (ericwolz) [inline README.md]: "that result on the top"  ???

**Review by ericwolz — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (danewalton-msft) [reply README.md]: should probably say "appear"

File: README.md (hunk 1 above)
Comment (danewalton-msft) [reply README.md:518]: The link yes. The general instructions no. We can update it once they submit theirs

**Pull request merged**